plugins/janus_streaming.c

```diff
@@ -911,8 +916,10 @@ typedef struct janus_streaming_buffer {
 
 typedef struct janus_streaming_rtp_source {
 	gint audio_port;
+	gint audio_rtcp_port;
 	in_addr_t audio_mcast;
 	gint video_port[3];
+	gint video_rtcp_port;
```

Comment:
i was wondering the same thing. a single RTP stream is a much easier test case, though. once this code is working for a single RTP stream it will be easy to expand it if we need to add two more ports.